src/types/controllerTypes.ts

```diff
@@ -42,6 +37,11 @@ export interface WcConnectionCtrlState {
   pairingError: boolean;
 }
 
+// -- ExplorerCtrl ------------------------------------------- //
```

Comment:
Non Blocking: What is the difference here?

Reply:
no difference, im not sure why i moved it